src/MICore/LaunchOptions.cs

```diff
@@ -1812,6 +1850,18 @@ protected void InitializeCommonOptions(Xml.LaunchOptions.BaseLaunchOptions sourc
             // Ensure that CoreDumpPath and ProcessId are not specified at the same time
             if (!String.IsNullOrEmpty(source.CoreDumpPath) && source.ProcessIdSpecified)
                 throw new InvalidLaunchOptionsException(String.Format(CultureInfo.InvariantCulture, MICoreResources.Error_CannotSpecifyBoth, nameof(source.CoreDumpPath), nameof(source.ProcessId)));
+
+            if (source.SymbolLoadInfo != null)
+            {
+                SymbolInfoLoadAll = source.SymbolLoadInfo.LoadAllSpecified ? source.SymbolLoadInfo.LoadAll : true;
+                SymbolInfoExceptionList.SetTo(source.SymbolLoadInfo.ExceptionList == null ? new string[0] : source.SymbolLoadInfo.ExceptionList.Split(';'));
+
+                // Ensure that symbol loading options are consistent
+                if (!WaitDynamicLibLoad && !SymbolInfoExceptionList.IsEmpty)
```

Comment:
>if [](start = 16, length = 2)

Since the LLDB command factory returns false for `SupportsStopOnDynamicLibLoad` should this be an error to include this with LLDB?

Reply:
yes. done